### DISCUSSION TITLE
Is it possible to set different prConcurrentLimits depending on update type?

### DISCUSSION THREAD

**Discussion by donleistman** [renovatebot/renovate#18133] [Request Help]
Hi there,

Wanted to clarify something that I could not find explicitly addressed in the docs. 

We have a large backlog of package updates. Using renovate for a while, we're now at the point where we have most of our package are on the latest minor / patch updates, and most of our backlog consists of major version updates.

We currently have the `prConcurrentLimit` rule set to `5` to avoid renovate opening PRs for the entire backlog. At this point, however, most of the major version updates tend to stay open for a while as they require developer intervention to address breaking changes. This eats up all our concurrent slots, so that new patch/minor updates cannot be opened/automerged. I've been triggering PRs manually from the dependency dashboard to keep minor/patch versions up-to-date, but would like this to be automated.

What I would like is for major version updates to remain limited to 5 open PRs, but to set a higher limit for minor/patch updates, so they will continue to be updated / automerged while a separate queue for major version updates remained filled.

Would the following be a valid configuration, and work as described above? Or does `prConcurrentLimit` only work at the top level of the config?

```json
{
  "extends": ["config:base"],
  "dependencyDashboard": true,
  "schedule": ["after 9am and before 6pm every weekday"],
  "timezone": "America/New_York",
  "prHourlyLimit": 3,
  "packageRules": [
    {
      "matchUpdateTypes": ["patch"],
      "automerge": true,
      "prConcurrentLimit": 10,
    },
    {
      "matchUpdateTypes": ["minor"],
      "matchCurrentVersion": "!/^0/",
      "automerge": true,
      "prConcurrentLimit": 10,
    },
    {
      "matchUpdateTypes": ["major"],
      "automerge": false,
      "prConcurrentLimit": 5,
    },
  ],
}

```

**Comment by viceice** [comment]
doesn't work. you should use dashboard approval for major updates, so they don't eat your PR limits.

**Comment by rarkins** [comment]
See #10546
